doc/programming_guide/media.rst

```diff
@@ -95,25 +119,102 @@ DirectSound is available only on Windows, and is installed by default.
 pyglet uses only DirectX 7 features. On Windows Vista, DirectSound does not
 support hardware audio mixing or surround sound.
 
+.. _guide-openal:
+
 OpenAL
 ^^^^^^
 
-OpenAL is included with Mac OS X. Windows users can download a generic driver
-from `openal.org`_, or from their sound device's manufacturer. Most Linux
-distributions will have OpenAL available in the repositories for download.
-For example, Arch users can ``pacman -S openal`` and Ubuntu users can ``apt install libopenal1``.
+By default, this is the most preferred driver. It will be used first
+if it is installed unless overridden. See :ref:`guide-audio-driver-order`
+to learn more.
 
-Pulse
-^^^^^
+It is the best target for cross-platform consistency due to being either
```

Comment:
I wouldn't necessary say it's the best for cross-platform, since XAudio2 (or even DirectSound) are fully featured with regards to positional audio. Instead, we _could_ say it's "currently the best choice for posix systems" (or just leave it out).